pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -63,9 +64,23 @@ def data_for_grouping():
 class BaseDecimal(object):
 
     def assert_series_equal(self, left, right, *args, **kwargs):
-
-        left_na = left.isna()
-        right_na = right.isna()
+        def convert(x):
+            # need to convert array([Decimal(NaN)], dtype='object') to np.NaN
```

Comment:
hmm, we *could* add this support I think though, create an issue?

Reply:
Right now we have a pretty clear answer of "decimal isn't supported". I'm hesitant to even partially support it :)